Resources/Prototypes/SS220/Entities/Clothing/Neck/BlueShieldCloak.yml

```
@@ -0,0 +1,8 @@
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingCloakBlueShield2
```

```suggestion
  id: ClothingCloakBlueShield2
```
Почему 2? у него же нет первого плаща?

---

у него нет, но оффы могут ввести.. чтобы не было проблемами с ID сразу дал ему измененный